.travis.yml

```diff
@@ -10,16 +10,20 @@ before_install:
 - sudo apt-get install --reinstall ca-certificates
 - sudo -E add-apt-repository ppa:beineri/opt-qt591-trusty -y
 - sudo apt-get update -qq
-- sudo apt-get install --force-yes -qq qt59base qt59tools libqt5scintilla2-dev
-- sudo ln -s /usr/include/qt5/Qsci /opt/qt59/include/Qsci
-- sudo ln -s /usr/share/qt5/mkspecs/features/qscintilla2.prf /opt/qt59/mkspecs/features/qscintilla2.prf     
+- sudo apt-get install --force-yes -qq qt59base qt59tools    
 
 script:
-  - source /opt/qt59/bin/qt59-env.sh
+  - source /opt/qt59/bin/qt59-env.sh 
+  - wget https://sourceforge.net/projects/pyqt/files/QScintilla2/QScintilla-2.10.1/QScintilla_gpl-2.10.1.tar.gz
+  - tar -xvzf QScintilla_gpl-2.10.1.tar.gz
+  - cd QScintilla_gpl-2.10.1/Qt4Qt5
+  - qmake "CONFIG+=release silent" 
+  - make
```

Comment:
Whan added -j parameter with the number of processors cores the program will crash.